packages/orchestration/src/exos/zcf-for-flow.js

```diff
@@ -0,0 +1,75 @@
+import { VowShape } from '@agoric/vow';
+
+/**
+ * @import {HostOf} from '@agoric/async-flow';
+ * @import {VowTools} from '@agoric/vow';
+ * @import {Zone} from '@agoric/zone';
+ */
+
+import { M } from '@endo/patterns';
+
+export const ZcfI = M.interface(
+  'ZCF',
+  {
+    makeInvitation: M.call(M.raw(), M.string())
```

Comment:
Given that the membrane does not support non-passable arguments, let's restrict to a handler in remotable form ?

```suggestion
    makeInvitation: M.call(M.remotable('OfferHandler'), M.string())
```